app/src/main/java/com/eveningoutpost/dexdrip/g5model/Ob1G5StateMachine.java

```diff
@@ -1707,11 +1707,12 @@ public static void evaluateG6Settings() {
             if (FirmwareCapability.isTransmitterG6(getTransmitterID())) {
                 if (!usingG6()) {
                     setG6Defaults();
-                    JoH.showNotification("Enabled G6", "G6 Features and default settings automatically enabled", null, Constants.G6_DEFAULTS_MESSAGE, false, true, false);
+                    JoH.showNotification("Enabled defaults", "Default settings automatically enabled", null, Constants.G6_DEFAULTS_MESSAGE, false, true, false);
                 } else if (!onlyUsingNativeMode() && !Home.get_engineering_mode()) {
                     // TODO revisit this now that there is scaling
                     setG6Defaults();
-                    JoH.showNotification("Enabled G6", "G6 Native mode enabled", null, Constants.G6_DEFAULTS_MESSAGE, false, true, false);
+                    UserError.Log.e(TAG, "Dex Native mode enabled.  For your device, non-native mode is either not possible or not recommended.");
```

Comment:
`Log.uel` or `Log.ueh` might be better than Log.e` for stand out informative messages

Reply:
Good suggestion.  Thanks
Done